webpack.common.js

```diff
@@ -0,0 +1,81 @@
+/* jshint esversion: 9 */
+
+const path = require('path');
```

Comment:
This file is hard to read with the eslint errors.  
Could you add a /* globals ...  */ statement to the top to silence those note defined  messages please?

Then have eslint installed (it is a node.js tool info is on the main README) and you can even do a eslint --fix to fix all the whitespace and indenting problems automagically.

Reply:
I'm pretty sure VScode is using eslint, and reports only one questionmark for this file -- the use of require rather than import. But the formatting of this file, even for Webpack 5, is still an "old-style" module that predates import/export.

Reply:
(Make sure that your eslint config, in turn, is not still making really old assumptions. There won't be ANY globals going forward. Modules have no globals!)

Reply:
I just deleted my comment about .eslintrc.js because I see it's actually pretty modern in its assumptions, now I wonder why you're seeing errors I don't. jshint thinks the file is clean for example.

Reply:
I think the problem is that .eslintrc.js isn't currently configured for Node.js files (which webpack's configs are). I'll look into the right settings.

Reply:
OK, I was clearly not configured to use eslint as I thought i was, so I'm bringing that inline and taking some of the recommendations into account.

One I want to change though: 80 characters is ridiculous in this day and age. I prefer 100. Thoughts?

Reply:
Also, the trailing-space thing was something that mystified me, because I thought I had configured VScode to strip them. Turns out that's only true on my work machine. Derp.